usrsctplib/netinet/sctputil.c

```diff
@@ -2162,6 +2162,7 @@ sctp_timeout_handler(void *t)
 		did_output = false;
 		break;
 	default:
+		did_output = false;
```

Comment:
What about initializing this at 1782 and let different cases to switch it as needed?

There is a similar warning at line 2609 (`to_ticks`).